workshop/style-parser/parse-ol.md

```diff
@@ -1,29 +1,29 @@
 
-# OpenLayers Stil parsen
+# Parsing an OpenLayers Style
 
-Einen OpenLayers Stil zu parsen erfordert die gleichen Schritte, wie sie auch beim parsen eines SLDs nötig sind.
-Wir werden wieder die `readStyle` und `writeStyle` Funktionen des Parsers verwenden. Diesmal machen wir allerdings
-vom `geostyler-openlayers-parser` gebrauch und benutzen keinen SLD String, sondern ein OpenLayer Stil Objekt.
+Parsing an OpenLayers style requires the same steps as parsing SLDs.
+We will again use the `readStyle` and `writeStyle` methods of the parsers. However, this time, we will use
+the `geostyler-openlayers-parser` and do not use an SLD string, but rather an OpenLayers style object.
 
-Die Installation des `geostyler-openlayers-parser` wurde bereits in einem vorherigen Kapitel mittels
+The installation of the `geostyler-openlayers-parser` was already done in a previous chapter via
```

Comment:
A link to the previous chapter may be helpful for readers.

Reply:
Since all content is rendered on a single page, I think this is not needed.